lunar/distribution.yaml

```diff
@@ -11,6 +11,11 @@ release_platforms:
   - xenial
   - zesty
 repositories:
+  Advanced_Navigation_driver:
```

Comment:
We don't officially have a policy for repository names, but please stick to the lowercase underscoring naming scheme. http://www.ros.org/reps/rep-0144.html